<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopResultDialog</class>
 <widget class="QDialog" name="TopResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonTopRefresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Refresh list</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewTopList">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTopList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCategory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Categories:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTopCategory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoToUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
